### ISSUE TITLE
News on twisted website

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8850]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| gravis reported|
|-|-|
|Trac ID|trac#406|
|Type|enhancement|
|Created|2003-11-19 21:32:44Z|

<details><summary>Searchable metadata</summary>

```
trac-id__406 406
type__enhancement enhancement
reporter__gravis gravis
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__ 
time__1069277564000000 1069277564000000
changetime__1469444904946961 1469444904946961
version__ 
owner__ 
cc__radix cc__spiv cc__gravis
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
Hi,

this would be cool to have some news items on the twisted website (which are
often on the irc channel topic). The home page seems too 'static' to me :(
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
A prerequisite for this would be for someone to write some news... are you
volunteering? :)

We actually had a news link until quite recently, but it got removed due to lack
of updates.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
I can do it :)
My english isn't perfect, so I'll need someone the correct the mistakes !
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
I'm here for you, pally
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
Thank you so much ! I'll propose some news this evening so :)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
real-time communication would be helpful for collaborating on news. Shall we get
together on IRC tonight? I assume you want to do summaries of mailing list
threads and suchlike -- we can discuss who will cover what topics.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="gravis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gravis commented|
|-|-|

```
#!html
<pre>
Of course we can, but I'm in France, and "tonight"is not the same time for me :)
where are you ?
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
resolving as WONTFIX. someone can repon this or open up a new bug if they want.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_10"></a>|GitHub commented|
|-|-|

In changeset 5b10ed051c6d0d04711b77179b65a8867797fe2e
```
#!CommitTicketReference repository="" revision="5b10ed051c6d0d04711b77179b65a8867797fe2e"
Merge pull request #406 from twisted/8663-namestaponwin

Author: hawkowl
Reviewers: adiroiban, rodrigc
Fixes: #8663
```
